command/tls_ca_create.go

```diff
@@ -163,3 +259,17 @@ func (c *TLSCACreateCommand) Run(args []string) int {
 
 	return 0
 }
+
+// IsEmpty checks whether any of TLSCACreateCommand parameters have been populated with
+// non-default values.
+func (c TLSCACreateCommand) IsEmpty() bool {
```

Comment:
Use a pointer otherwise `TLSCACreateCommand` has methods on both value and pointer receivers.
```suggestion
func (c *TLSCACreateCommand) IsEmpty() bool {
```

This function name is also a little misleading, as it doesn't check whether the command is totally empty, just certain parameters.